apps/src/applab/applab.js

```diff
@@ -67,6 +75,13 @@ var jsInterpreterLogger = null;
 var debuggerUi = null;
 
 /**
+ * Redux Store holding application state, transformable by actions.
+ * @type {Store}
+ * @see http://redux.js.org/docs/basics/Store.html
+ */
+Applab.reduxStore = createStore(rootReducer);
```

Comment:
Curious—design-wise is it the case that applab & gamelab's `AppName.store`s might move upwards to `StudioApp.prototype` or somewhere else at some point in the future?